client/maitenance_test.go

```diff
@@ -35,17 +34,19 @@ func TestClientFactoryReset(t *testing.T) {
 		},
 	}
 
-	c := NewTestClient()
+	c, err := NewTestSecureClient()
+	require.NoError(t, err)
 	defer func() {
 		err := c.Close(context.Background())
 		require.NoError(t, err)
 	}()
-
+	ctx, cancel := context.WithTimeout(context.Background(), TestTimeout)
+	defer cancel()
+	deviceID, err = c.OwnDevice(ctx, deviceID)
+	require.NoError(t, err)
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			ctx, cancel := context.WithTimeout(context.Background(), time.Second*1)
-			defer cancel()
-			err := c.FactoryReset(ctx, tt.args.deviceID)
+			err := c.FactoryReset(ctx, deviceID)
```

Comment:
tt.args.deviceID? Rr the second test case must be modified since it uses different deviceID value

Reply:
I want to test it with non exist device so will leave it and use proper `tt.args.deviceID`